packages/xo-web/src/xo-app/hub/recipes/recipe.js

```diff
@@ -0,0 +1,118 @@
+import * as ComplexMatcher from 'complex-matcher'
+import _ from 'intl'
+import ActionButton from 'action-button'
+import ButtonLink from 'button-link'
+import decorate from 'apply-decorators'
+import Icon from 'icon'
+import marked from 'marked'
+import React from 'react'
+import { Card, CardBlock, CardHeader } from 'card'
+import { escapeRegExp } from 'lodash'
+import { form } from 'modal'
+import { connectStore } from 'utils'
+import { createGetObjectsOfType } from 'selectors'
+import { createKubernetesCluster } from 'xo'
+import { injectState, provideState } from 'reaclette'
+import { success } from 'notification'
+import { withRouter } from 'react-router'
+
+import RecipeForm from './recipe-form'
+
+const RECIPE_INFO = {
+  id: '05abc8a8-ebf4-41a6-b1ed-efcb2dbf893d',
+  name: 'Kubernetes cluster',
+  description:
+    'Creates a Kubernetes cluster composed of 1 master and a configurable number of nodes working for the master.',
+}
+
+export default decorate([
+  withRouter,
+  connectStore(() => ({
+    pools: createGetObjectsOfType('pool'),
+    recipeCreatingResources: state => state.recipeCreatingResources,
+  })),
+  provideState({
+    initialState: () => ({
+      selectedInstallPools: [],
+    }),
+    effects: {
+      async create() {
+        const { markRecipeAsCreating, markRecipeAsDone } = this.props
+        const recipeParams = await form({
+          defaultValue: {
+            pool: {},
+          },
+          render: props => <RecipeForm {...props} />,
+          header: (
+            <span>
+              <Icon icon='hub-recipe' /> {RECIPE_INFO.name}
+            </span>
+          ),
+          size: 'medium',
+        })
+
+        const {
+          cidr,
+          masterName,
+          nbNodes,
+          network,
+          pool,
+          sr,
+          sshKey,
+        } = recipeParams
+
+        markRecipeAsCreating(RECIPE_INFO.id)
+        const tag = await createKubernetesCluster({
+          cidr,
+          masterName,
+          nbNodes: +nbNodes,
+          network: network.id,
+          pool: pool.id,
```

Comment:
Please remove this param (due to [this change](https://gitlab.com/vates/xoa/merge_requests/44/diffs#1a351ba5e40bfa791471a38313bbe607da21460e_185_185)).

Reply:
Done :heavy_check_mark: